README.md

```diff
@@ -43,43 +43,79 @@ zmu supports Linux and Windows operating systems.
 - System Simulation:
     - device profiles, eg stm32 device support
     - board profiles, external peripheral simulation
+
+## Depedencies
+
+You need ARM compiler. Get the latest compiliers from [ARM website](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads)
+
+You have to install RUST. Assuming Ubuntu 18.04.
+
+```sh
+curl https://sh.rustup.rs -sSf | sh
+```
+
+Follow the install menu, then run the following command in the terminal used for compilation
+
+```sh
+source ~/.cargo/env
+```
+
+## How to Compile
```

Comment:
In strict sense the compilation does not require GCC, just the specific examples (to show that simple GCC compiled binaries can be emulated). As there is separate instructions for compiling and running the testsuite, maybe the GCC_HOME could be moved to the testsuite running part?

Reply:
understood ! perhaps this section should be called 'How to compile the examples'. Feel free to move to the testsuite part.